source/manual/ruby.html.md

```diff
@@ -32,17 +32,37 @@ We set up rbenv differently depending on what's going on:
 
 ## Add a new Ruby version in puppet
 
-Build a package using [fpm](debian-packaging.html#fpm) and make it
-available using our aptly repository.
+You will need to build a new [fpm](debian-packaging.html#fpm) package with the new Ruby version.
+This package can then be copied to Aptly machine, and the new version added to puppet.
 
-The package name includes the version of Ruby (eg `rbenv-ruby-2.3.1`) because
-we need to install multiple versions of Ruby at the same time.
+### Building the fpm package.
 
-Once it's available as a package you can
+- Add a new recipe for the ruby version in [Packager](packager).
+The folder name will be the Ruby version, and contain a `recipe.rb` file. See previous entries for examples.
+The recipe will require the [SHA256](sha256_checksum) of the version's `tar.gz`, available at [Ruby cache](ruby_cache).
+
+- Once the Packager change is merged, [build the package](jenkins).
```

Comment:
```suggestion
- Once the Packager change is merged, [build the package][jenkins].
```

Reply:
Thanks @thomasleese ! Fixed this in [cc0d8c5](https://github.com/alphagov/govuk-developer-docs/commit/cc0d8c5c2c5122a18a30bca911ab6470ec70218a)